shared/templates/static/ansible/accounts_root_path_dirs_no_write.yml

```diff
@@ -0,0 +1,25 @@
+# platform = multi_platform_rhel, multi_platform_fedora
+# reboot = false
+# strategy = restrict
+# complexity = low
+# disruption = medium
+
+- name: "Fail if user is not root"
+  fail:
+    msg: 'Root account required to read root $PATH'
+  when: ansible_user != "root"
```

Comment:
@ybznek shouldn't all the ansible remediations check if the `ansible_user` is `root`?